client/my-sites/checkout/src/components/checkout-main-content.tsx

```diff
@@ -352,7 +357,21 @@ export default function CheckoutMainContent( {
 		removeCoupon,
 		couponStatus,
 	} = useShoppingCart( cartKey );
-	const translate = useTranslate();
+
+	const searchParams = new URLSearchParams( window.location.search );
+	const isDIFMInCart = hasDIFMProduct( responseCart );
+	const isSignupCheckout = searchParams.get( 'signup' ) === '1';
+	const selectedSiteData = useSelector( getSelectedSite );
+	const wpcomDomain = useSelector( ( state ) =>
+		getWpComDomainBySiteId( state, selectedSiteData?.ID )
+	);
+
+	/*
+	 * Only show the site preview for WPCOM domains that have a site connected to the site id
+	 * */
```

Comment:
You have an extra asterisk here, which got me curious about Calypso coding standards for JS comments. [It turns out that we prefer `//` for multi-line, non-JSDoc comments.](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/javascript.md#comments)

Reply:
Thanks for the clarification, updating.